app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,7 @@
     <string name="curr_speed_label">Current</string>
     <string name="avg_speed_label">Average</string>
     <string name="initial_speed">0.0 m/s</string>
+    <string name="TitleLocation">Activate location</string>
```

Comment:
I think the string names should be snake_case not CamelCase, but it's not that important

Reply:
I let like this for the moment